packages/hub/tasks/notify-customer-payment.ts

```diff
@@ -65,9 +69,23 @@ export default class NotifyCustomerPayment {
       );
     }
 
+    let merchantName = '';
+
+    try {
+      if (result.merchantSafe?.infoDid) {
+        let merchantInfo = await this.merchantInfo.getMerchantInfo(result.merchantSafe.infoDid);
+
+        if (merchantInfo) {
+          merchantName = ` ${merchantInfo.name}`;
+        }
+      }
+    } catch (e) {
+      // Silently ignore and just show notification without merchant name
```

Comment:
Is it worth logging this to Sentry?

Reply:
Maybe? I’m wary of more noisy logging, but perhaps an error here is worth knowing about. I’ll do the same in the other task, thanks.

Reply:
done in c241690 🤓